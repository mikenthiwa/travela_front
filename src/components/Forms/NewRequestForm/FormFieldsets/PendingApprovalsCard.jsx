import React, { Component } from 'react';

export default class PendingApprovals extends Component {
  render() {
    return (
      <div className="mdl-cell mdl-cell--5-col mdl-cell--12-col-tablet mdl-cell--12-col-phone">
        <div className="pending-approvals-rectangle">
          <div className="pending-approvals-text">
            <p> Pending Approvals For This Request</p>
          </div>
          <hr className="pending-approvals-line" />
          <div className="pending-approvals-block">
            <ul className="approval-list-items">
              <li className="approval-item"> 
                <div className="oval" />
                <div>Line Manager Approval</div>
              </li>
              <li className="approval-item"> 
                <div className="oval" />
                <div>Budget Checker Approval</div>
              </li>
              <li className="approval-item"> 
                <div className="oval" />
                <div>Travel readiness Verification</div>
              </li>
            </ul>
          </div>
        </div>
      </div>
    );
  }
}
