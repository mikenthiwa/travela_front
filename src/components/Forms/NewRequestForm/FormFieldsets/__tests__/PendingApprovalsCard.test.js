import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import PendingApprovalsCard from '../PendingApprovalsCard';


describe('<PendingApprovalsCard />',() => {
  const wrapper = mount(
    <MemoryRouter>
      <PendingApprovalsCard />
    </MemoryRouter>
  );
  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
