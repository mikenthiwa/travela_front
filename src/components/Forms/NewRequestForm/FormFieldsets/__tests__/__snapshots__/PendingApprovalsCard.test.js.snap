// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PendingApprovalsCard /> renders without crashing 1`] = `
<div
  className="mdl-cell mdl-cell--5-col mdl-cell--12-col-tablet mdl-cell--12-col-phone"
>
  <div
    className="pending-approvals-rectangle"
  >
    <div
      className="pending-approvals-text"
    >
      <p>
         Pending Approvals For This Request
      </p>
    </div>
    <hr
      className="pending-approvals-line"
    />
    <div
      className="pending-approvals-block"
    >
      <ul
        className="approval-list-items"
      >
        <li
          className="approval-item"
        >
          <div
            className="oval"
          />
          <div>
            Line Manager Approval
          </div>
        </li>
        <li
          className="approval-item"
        >
          <div
            className="oval"
          />
          <div>
            Budget Checker Approval
          </div>
        </li>
        <li
          className="approval-item"
        >
          <div
            className="oval"
          />
          <div>
            Travel readiness Verification
          </div>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
